PostHogObjCExample/AppDelegate.m

```diff
@@ -29,6 +29,14 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 
     PostHogConfig *config = [[PostHogConfig alloc] apiKey:@"_6SG-F7I1vCuZ-HdJL3VZQqjBlaSb1_20hDPwqMNnGI"];
     config.preloadFeatureFlags = YES;
+    config.sessionReplayConfig.captureLogs = YES;
```

Comment:
does that work on the swift samples as well? can we enable in the ios swift example?